package.json

```diff
@@ -101,6 +101,7 @@
     "eslint-plugin-ember-internal": "^1.1.0",
     "express": "^4.15.2",
     "finalhandler": "^1.0.2",
+    "fs-extra": "^4.0.1",
```

Comment:
I am confused. You said we are using this in blueprints, wouldn't this need to be in `dependencies` not `devDependencies`?

Reply:
That's wierd. I don't know what I was thinking. Will fix.